packages/e2e-test-app/test/TextComponentTest.test.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import {goToComponentExample, dumpVisualTree} from './framework';
+
+beforeAll(async () => {
+  await goToComponentExample('Text');
+});
+
+describe('Text examples dump test', () => {
+  test('text_dump_test', async () => {
```

Comment:
nit: The style I usually see for jest tests is to use spaces between words instead of underscore or hyphen.